tests/phpunit/tests/rest-api/rest-tags-controller.php

```diff
@@ -1502,4 +1508,91 @@ protected function check_get_taxonomy_term_response( $response, $id ) {
 		$tag  = get_term( $id, 'post_tag' );
 		$this->check_taxonomy_term( $tag, $data, $response->get_links() );
 	}
+
+	/**
+	 * @dataProvider data_readable_http_methods
+	 * @ticket 56481
+	 *
+	 * @param string $method HTTP method to use.
+	 */
+	public function test_get_items_returns_only_fetches_ids_for_head_requests( $method ) {
```

Comment:
```suggestion
	public function test_get_items_returns_only_ids_for_head_requests( $method ) {
```
Suggestion per above.

Reply:
https://github.com/WordPress/wordpress-develop/pull/7970#discussion_r1876900479